pkg/command/service/measure.go

```diff
@@ -626,21 +626,22 @@ kperf service measure --svc-perfix svc --range 1,200 --namespace ns --concurrenc
 				fmt.Printf("Percentile99: %fs\n", measureFinalResult.Result.P99)
 
 				current := time.Now()
-				rawPath := fmt.Sprintf("/tmp/%s_%s", current.Format("20060102150405"), "raw_ksvc_creation_time.csv")
+				outputLocation := measureArgs.output
+				rawPath := fmt.Sprintf("%s/%s_%s", outputLocation, current.Format("20060102150405"), "raw_ksvc_creation_time.csv")
```

Comment:
Why not have option for file name?

Reply:
@maximilien Thanks for your comment! There are 4 files will be generated into the output directory, so the design is end-user can specify the output directory, but the files will use auto-generated files.

Reply:
@ShiningTian @maximilien 
I assume the user scenario is end-user will check the html dashboard for investigation. The csv files are various kind of raw data to generate the html file. And users may be confused about what they are.

So I think we 1) keep the auto-generated file name but let users to specify the directory.
2) If needed we should redesign the result files (maybe in another PR)

Reply:
currently let's choose the option 1)

Reply:
OK but I guess you should include timestamp in file name for multiple runs?

Reply:
@maximilien make sense, just added a new issue https://github.com/knative-sandbox/kperf/issues/60